test/test_tensor_creation_ops.py

```diff
@@ -3487,6 +3487,22 @@ def test_randint_inference(self, device):
             self.assertIs(torch.int64, torch.randint(*args, size=size, out=out).dtype)
             self.assertIs(torch.int64, torch.randint(*args, size=size, out=out, dtype=torch.int64).dtype)
 
+        self.assertRaisesRegex(RuntimeError,
```

Comment:
should we also test with `torch.compile`?

Reply:
It is automatic tested if run with `PYTORCH_TEST_WITH_DYNAMO=1`, and I guess it is enabled in the CI? As some other test cases are marked with `xfailIfTorchDynamo` in the same file.

https://github.com/pytorch/pytorch/blob/bd019c0bb485904a99fb38589444b1461ab1e486/test/test_tensor_creation_ops.py#L1697-L1699